docs/_embed/benefits-embedding.md

```diff
@@ -10,7 +10,11 @@ redirect_from:
 # Benefits of embedding Zapier
 
 ## Introduction
-We’ve put together a list of opportunities to help your integration become more visible. Try one or all of the options here and begin connecting your users with Zapier’s 6000+ integrations.
+We’ve put together a list of opportunities to help your integration become more visible. Try one or all of the options here and begin connecting your users with Zapier’s 6, 000+ integrations.
+
+## Allow Users to Connect Faster through accelerated sign-ups
```

Comment:
Allow users to connect faster through accelerated sign-ups

([style guide](https://coda.io/d/Zapier-Content-and-Messaging-Guide_dH5LiS8Mxyt/Style-and-mechanics_suRp5#_lu_D6) is sentence case for headings)